app/controllers/friendships_controller.rb

```diff
@@ -24,13 +23,19 @@ def show; end
 
   def update
     @friendship = Friendship.find(params[:id])
-    redirect_to users_path if (@friendship.confirmed == false) && @friendship.accept_request
+    @friendship.confirmed = true
+
+    if @friendship.save
+      @friendship.confirm_friend
+      redirect_to user_path(current_user.id), notice: 'Friend request was successfully confirmed.'
+    else
+      redirect_to user_path(current_user.id), alert: @friendship.error.full_messages.join('. ').to_s
+    end
```

Comment:
- [ ] There is currently no way to invite other users to friendship and confirm or reject friendships. This is the screenshot that is displayed. Note that I haven't invited any user to friendship. This get's displayed by default. Clicking the `Friend Requests` link also does not work. Please, let's implement the features as specified in the project requirement including timeline posts.

![Screenshot 2021-09-16 at 23-40-12 RorSocialScaffold](https://user-images.githubusercontent.com/70150423/133702332-73ed113e-ea09-4390-89b1-496c08850d7b.png)

- [ ] Somehow, the posts are not getting displayed when a user creates a new post. Please, let's fix this as well

![Screenshot 2021-09-16 at 23-46-56 RorSocialScaffold](https://user-images.githubusercontent.com/70150423/133702665-8f0edf7a-8afe-4702-b50b-8c3685d32bcc.png)